src/main/java/com/google/devtools/build/lib/metrics/MetricsCollector.java

```diff
@@ -156,6 +157,16 @@ public void onActionComplete(ActionCompletionEvent event) {
   @AllowConcurrentEvents
   public void actionResultReceived(ActionResultReceivedEvent event) {
     spawnStats.countActionResult(event.getActionResult());
+    ActionStats actionStats =
+            actionStatsMap.computeIfAbsent(event.getAction().getMnemonic(), ActionStats::new);
+    Duration systemTime = event.getActionResult()
+            .cumulativeCommandExecutionSystemTime()
+            .orElse(Duration.ZERO);
```

Comment:
It is potentially quite misleading that the times can silently be zero if the ActionResult doesn't have any times on it.

Reply:
I have pushed the update addressing @larsrc-google's concern. What do you think about my latest update?